### PR TITLE
Added dependency check script

### PR DESCRIPTION
**Pull request by mukkachaitanya**
### Requirements

* Need for managing the outdated dependencies easily.

### Description of the Change

Adds `dependency-check` script to take care of outdated dependencies. 

### Benefits

Know and update outdated dependencies faster. 

### Applicable Issues

Addresses #107

### CODE REVIEW

**Comment by prasadtalasila** [comment]
@mukkachaitanya Suprised that I did not receive notification regarding your update on this PR. Two suggestions:
1. Updates to travis setup
    Please update the travis build environment to Ubuntu Xenial 16.04. For an example, see [.travis.yml](https://github.com/AutolabJS/AutolabJS/blob/dev/.travis.yml). The travis environment also allows us to select a version of Node.js using nvm. See the [example](https://github.com/AutolabJS/AutolabJS/blob/dev/.nvmrc). If not already done, it is time to update your local development environment to the Node.JS 10.x LTS.
1.  The travis build [log](https://travis-ci.org/AutolabJS/autolabcli/builds/474545839#L748) shows a few unused dependencies: `codeclimate-test-reporter`, `coveralls` and `mock-stdin`. If these dependencies are not being used, please remove them from package.json and update the package-lock.json.

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
+dist: xenial
+
 language: node_js
-node_js:
-  - "8"
 
 install:
   - npm install
```
Comment (prasadtalasila) [inline .travis.yml]: @mukkachaitanya The npm gets installed and managed via `nvm`.

Comment (mukkachaitanya) [reply .travis.yml]: So, sir, I need that parameter?

Comment (prasadtalasila) [reply .travis.yml]: What I mean is that the `npm install` command may not be required at all.

Comment (mukkachaitanya) [reply .travis.yml]: Ok. Understood sir.

**Pull request merged**